src/metabase/models/setting.clj

```diff
@@ -1281,6 +1287,15 @@
                (when api/*is-superuser?*
                  [:admin]))))
 
+(defn- filtered-user-facing-settings
```

Comment:
Not thrilled with this name

Reply:
Just making sure - what's the purpose of this change? Is this just a refactoring?

Reply:
Yeah it's incidental, and happy to back it out. Made the change earlier when I'd anticipate changing `admin-writable-site-wide-settings` in some intrusive way, but ended up not going that route.

 I still think it's nice to DRY it up so just left it in.

Reply:
I'm thinking that maybe it should be `make-user-facing-settings` or something in that vein? It's not only filtering them.

Reply:
Good point, I'll iterate on the name to reflect the transformation